rest/route/service.go

```diff
@@ -167,7 +167,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/projects/{project_id}/task_stats").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectTaskStats(opts.URL))
 	app.AddRoute("/projects/{project_id}/test_stats").Version(2).Get().Wrap(requireUser, viewTasks, cedarTestStats).RouteHandler(makeGetProjectTestStats(opts.URL))
 	app.AddRoute("/projects/{project_id}/versions").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectVersionsHandler(opts.URL))
-	app.AddRoute("/projects/{project_id}/tasks/{task_id}").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectTasksHandler(opts.URL))
+	app.AddRoute("/projects/{project_id}/tasks/{task_name}").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectTasksHandler(opts.URL))
```

Comment:
Who is currently using this route? Is it heavily used by users? If yes, we may want to make this change in a backwards compatible way (maybe still set h.taskName with both gimlet.GetVars(r)["task_name"] and gimlet.GetVars(r)["task_id"]

Reply:
I don't think this should effect users, since this is just related to how we parse the URL and it's not something that users set (for example, "/projects/evg/tasks/dist" should work either way )